apps/rush-lib/src/cli/actions/CustomRushAction.ts

```diff
@@ -175,27 +154,44 @@ export class CustomRushAction extends BaseRushAction {
       });
     }
 
-    // @TODO we should throw if they are trying to overwrite built in flags
+    // Find any parameters that are associated with this command
+    for (const parameter of this._commandLineConfiguration.parameters) {
+      let associated: boolean = false;
+      for (const associatedCommand of parameter.associatedCommands) {
```

Comment:
>associatedCommands [](start = 48, length = 18)

We should validate that every associated command corresponds to a defined command.